src/errorpage.cc

```diff
@@ -206,6 +207,16 @@ static MemBuf error_stylesheet;
 static const char *errorFindHardText(err_type type);
 static IOCB errorSendComplete;
 
+/// \ingroup ErrorPageInternal
+static const char *
+FindDefaultTemplate(const err_type type)
+{
+    const auto foundDefault = std::find_if(begin(TemplateDefaults), end(TemplateDefaults), [&](const DefaultMessages &m) {
+        return m.type == type;
+    });
+    return foundDefault == end(TemplateDefaults) ? nullptr : foundDefault->text;
+}
```

Comment:
If possible, please use std::array::cbegin() and std::array::cend() methods instead of these std::begin() and std::end() globals. I think that would look more consistent with the rest of Squid code that uses similar expressions for STL containers.

Reply:
Done.